README.md

```diff
@@ -17,7 +17,7 @@ Currently, we have the following extensions:
 - [salesforcedx-vscode-apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex)  
    This extension uses the Apex Language Server to provide features such as syntax highlighting and code completion.
 - [salesforcedx-vscode-apex-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger)  
-   This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs.
+   This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs or to use ISV Customer Debugger.
```

Comment:
You use "and" instead of "or" in example just below. Can they use both at once or either?

Reply:
See my next comment.